modules/reindex/src/test/java/org/elasticsearch/reindex/AsyncBulkByScrollActionTests.java

```diff
@@ -335,6 +342,48 @@ public void testBulkResponseSetsLotsOfStatus() throws Exception {
         }
     }
 
+    public void testHandlesBulkWithNoScroll() {
+        // given a request that should not open scroll
+        testRequest.setMaxDocs(1);
```

Comment:
Can we use random 1-100?

```suggestion
        testRequest.setMaxDocs(between(1, 100));
```